compiler/z/codegen/OMRTreeEvaluatorTable.hpp

```diff
@@ -122,7 +122,6 @@
    TR::TreeEvaluator::dmulEvaluator,        // TR::dmul
    TR::TreeEvaluator::unImpOpEvaluator,        // TR::bmul
    TR::TreeEvaluator::unImpOpEvaluator,        // TR::smul
-   TR::TreeEvaluator::imulEvaluator,        // TR::iumul
```

Comment:
With the removal of unsigned multiplications some paths in the evaluator now become dead code which we should also remove. For example, once this PR is merged then [1] becomes dead code. Do we plan on performing such cleanup once full deprecation is done?

I think we need to open up an issue to track such work. We can take a look at which unsigned opcodes we deprecated and look for `TR::Node::isUnsigned()` paths in the respective evaluators and remove all the dead code that results. Could you open such an issue?

[1] https://github.com/eclipse/omr/blob/1bc0b8512f7f192f377126b4cd56cf425901a5a0/compiler/z/codegen/BinaryEvaluator.cpp#L2669-L2692